apps/ledger-live-desktop/src/renderer/components/TranslatedError/TranslatedError.tsx

```diff
@@ -0,0 +1,97 @@
+// Convention:
+// - errors we throw on our app will use a different error.name per error type
+// - an error can have parameters, to use them, just use field of the Error object, that's what we give to `t()`
+// - returned value is intentially not styled (is universal). wrap this in whatever you need
+
+import React, { useEffect, useMemo } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import logger from "~/renderer/logger";
+import Text from "../Text";
+import ExternalLink from "../ExternalLink";
+import { openURL } from "~/renderer/linking";
+import { urls } from "~/config/urls";
+import { useErrorLinks } from "./hooks/useErrorLinks";
+
+type Props = {
+  error: Error | undefined | null;
+  field?: "title" | "description" | "list";
+  noLink?: boolean;
+  fallback?: React.ReactNode;
+};
+
+type ErrorListProps = {
+  translation: Object;
+};
+
+function ErrorList({ translation }: ErrorListProps) {
+  return (
+    <>
+      {Object.entries(translation).map(([key, str]) =>
+        typeof str === "string" ? <li key={key}>{str}</li> : null,
+      )}
+    </>
+  );
+}
+
+export function TranslatedError({ error, fallback, field = "title", noLink }: Props): JSX.Element {
+  const { t } = useTranslation();
+
+  const errorName = error?.name;
+
+  const translationKey = useMemo(() => `errors.${errorName}.${field}`, [errorName, field]);
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Do we need this? can we add the dep if it is missed?

Reply:
Yes, can be removed 👍